README.md

```diff
@@ -109,14 +120,57 @@ where data in form of snapshots can be exported to, and imported from.
 ```sql
 -- A policy using the SNAPSHOT strategy.
 INSERT INTO "ext"."retention_policy"
-  (table_schema, table_name, partition_column, retention_period, target_repository_name, strategy)
+  (strategy, table_schema, table_name, partition_column, retention_period, target_repository_name)
 VALUES
-  ('doc', 'sensor_readings', 'time_month', 365, 'export_cold', 'snapshot');
+  ('snapshot', 'doc', 'sensor_readings', 'time_month', 365, 'export_cold');
 ```
 
 [implementation](cratedb_retention/strategy/snapshot.py) | [tutorial](https://community.crate.io/t/building-a-data-retention-policy-for-cratedb-with-apache-airflow/1001)
 
 
+## Internals
+
+The database schema for the retention policy table is defined like this.
+Each record represents a rule how to expire and retain data per table.
+At runtime, each record will correspond to a dedicated retention job task.
+
+```sql
+-- Set up the retention policy database table schema.
+CREATE TABLE IF NOT EXISTS {policy_table.fullname} (
+
+    -- Source: The database table operated upon.
+    "table_schema" TEXT,                        -- The source table schema.
+    "table_name" TEXT,                          -- The source table name.
+    "partition_column" TEXT NOT NULL,           -- The source table column name used for partitioning.
+
+    -- Retention parameters.
+    "retention_period" INTEGER NOT NULL,        -- Retention period in days.
+                                                -- TODO: Provide better description what this means.
+
+    -- Target: Where data is moved/relocated to.
+    "reallocation_attribute_name" TEXT,         -- TODO: Describe the meaning of this column value.
```

Comment:
That is the name of the [node-specific custom attribute](https://crate.io/docs/crate/reference/en/5.3/config/node.html#custom-attributes) to identify target nodes to reallocate data to

Reply:
Thank you. I've adjusted the patch correspondingly.